### PR TITLE
docs: add docs for additional azs and container insights

### PR DESCRIPTION
**Pull request by efekarakus**
🙈

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/commands/env-init.en.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ You create environments using a [named profile](../credentials.en.md#environment
 Like all commands in the AWS Copilot CLI, if you don't provide required flags, we'll prompt you for all the information we need to get you going. You can skip the prompts by providing information via flags:
 ```
 Common Flags
+  -a, --app string                     Name of the application. (default "demo")
```
Comment (huanjani) [inline site/content/docs/commands/env-init.en.md]: nit: delete this bc specific to the workspace's app name
```suggestion
  -a, --app string                     Name of the application.
```

File: site/content/docs/commands/env-init.en.md [hunk 2]
```diff
@@ -29,26 +29,44 @@ Import Existing Resources Flags
...
 
 ## Examples
-Creates a test environment in your "default" AWS profile using default config.
+Creates a test environment using your "default" AWS profile and default configuration.
 ```bash
 $ copilot env init --name test --profile default --default-config
 ```
 
-Creates a prod-iad environment using your "prod-admin" AWS profile using existing VPC.
+Creates a prod-iad environment using your "prod-admin" AWS profile and enables container insights.
```
Comment (huanjani) [inline site/content/docs/commands/env-init.en.md]: We should probably capitalize "Container Insights" in the help menu, maybe even "CloudWatch Container Insights"?

File: site/content/docs/developing/custom-environment-resources.en.md [hunk 3]
```diff
@@ -16,7 +17,36 @@ Which credentials would you like to use to create name? [profile default]
...
+You may use the import feature to bring a VPC with only two private subnets and no public subnets for your workloads that are not internet-facing. (For more details on the resources you'll need for isolated networks, go [here](https://github.com/aws/copilot-cli/discussions/2378).)
+
+## Modifying Copilot's default resources 
+When you select the default configuration, Copilot follows [AWS best practices](https://aws.amazon.com/blogs/containers/amazon-ecs-availability-best-practices/) and creates a VPC with two public and two private subnets, with one of each type in one of two Availability Zones. 
+If you require additional availability zones or need to modify the CIDR ranges, you can opt-in to modify these settings:
```
Comment (huanjani) [inline site/content/docs/developing/custom-environment-resources.en.md]: nit: for the verb form
```suggestion
If you require additional availability zones or need to modify the CIDR ranges, you can opt in to modify these settings:
```

File: site/content/docs/commands/env-init.en.md [hunk 4]
```diff
@@ -29,26 +29,44 @@ Import Existing Resources Flags
...
+      --override-private-cidrs strings   Optional. CIDR to use for private subnets.
+                                         (default 10.0.2.0/24,10.0.3.0/24)
+      --override-public-cidrs strings    Optional. CIDR to use for public subnets. 
+                                         (default 10.0.0.0/24,10.0.1.0/24)
+      --override-vpc-cidr ipNet          Optional. Global CIDR to use for VPC.
+                                         (default 10.0.0.0/16)
 
-Global Flags
-  -a, --app string   Name of the application.
+Telemetry Flags
```
Comment (Lou1415926) [inline site/content/docs/commands/env-init.en.md:41]: Boss what do you think about "Observability" 👀 ?

File: site/content/docs/developing/custom-environment-resources.en.md [hunk 5]
```diff
@@ -1,8 +1,9 @@
 # Custom Environment Resources
 
+## Importing existing resources
 When creating a new [environment](../concepts/environments.en.md) with Copilot, you are given the option to import existing VPC resources. (Use [flags with `env init`](../commands/env-init.en.md#what-are-the-flags) or the guided experience, shown below.)
 ```bash
-% copilot env init
```
Comment (Lou1415926) [inline site/content/docs/developing/custom-environment-resources.en.md:5]: :homer-disappearing

File: site/content/docs/developing/custom-environment-resources.en.md [hunk 6]
```diff
@@ -16,7 +17,36 @@ Which credentials would you like to use to create name? [profile default]
   > No, I'd like to import existing resources (VPC, subnets).
 ```
 
-When you select the default configuration, Copilot follows [AWS best practices](https://aws.amazon.com/blogs/containers/amazon-ecs-availability-best-practices/) and creates a VPC with two public and two private subnets, with one of each type in one of two Availability Zones. While this is a good configuration for most cases, Copilot allows some flexibility when you import your own resources. For example, you may bring a VPC with only two private subnets and no public subnets for your workloads that are not internet-facing. (For more details on the resources you'll need for isolated networks, go [here](https://github.com/aws/copilot-cli/discussions/2378).)
+You may use the import feature to bring a VPC with only two private subnets and no public subnets for your workloads that are not internet-facing. (For more details on the resources you'll need for isolated networks, go [here](https://github.com/aws/copilot-cli/discussions/2378).)
```
Comment (Lou1415926) [inline site/content/docs/developing/custom-environment-resources.en.md:20]: Would this paragraph be better suited under the title "## Modifying Copilot's default resources"?

File: site/content/docs/commands/env-init.en.md (hunk 4 above)
Comment (efekarakus) [reply site/content/docs/commands/env-init.en.md:41]: ahaha yea, I want to rename it to that but the field in SSM and that appears in `copilot env init --json` is `"telemetry"` in v1.14 💀 so I don't think we can

File: site/content/docs/developing/custom-environment-resources.en.md (hunk 6 above)
Comment (efekarakus) [reply site/content/docs/developing/custom-environment-resources.en.md:20]: oh I think the disabling public subnets is only possible with the "Import VPC" feature and not while adjusting our default settings, so I kept this as is. Let me know if I'm missing something!

**Review by Lou1415926 — APPROVED** [review]

**Review by huanjani — APPROVED** [review]
Looks good! Not sure if you wanted to address the capitalization of "container insights" comment, so adding the DNM label.

**Pull request merged**